src/components/Navigation/index.tsx

```diff
@@ -0,0 +1,34 @@
+import { NavigationStyled, Groups, Profile, AddExpense } from './index.styled';
+import { useNavigate } from 'react-router-dom';
+import groups from '../../assets/images/groups-icon.svg'
+import receipt from '../../assets/images/receipt-icon.svg'
+import avatar from '../../assets/images/avatar-icon.svg'
+import { ROUTES } from '../../constants/routes';
+
+interface Props {
+  visible: boolean;
+}
+
+const Navigation = ({ visible }: Props) => {
+  const navigate = useNavigate();
+
+  if (visible)
+    return (
+      <NavigationStyled>
+        <Groups onClick={() => navigate(ROUTES.GROUPS.ROOT)}>
+          <img src={groups} alt="Groups" />
+        </Groups>
+
+        <AddExpense>
+          <img src={receipt} alt="AddExpense" />
+        </AddExpense>
+
+        <Profile>
+          <img src={avatar} alt="Profile" />
+        </Profile>
+      </NavigationStyled>
+    );
+  return null;
+}
+
+export default Navigation;
```

Comment:
Maybe it wold be better to call this BottomNavigation or FooterNavigation, don't you think?
Seeing just 'Navigation' makes me think about other associations... something more global maybe